features/machine/autoscaler.feature

```diff
@@ -179,3 +179,129 @@ Feature: Cluster Autoscaler Tests
     Then the step should succeed
     And the output should match "Annotations:\s+<none>"
 
+# @author zhsun@redhat.com
+  # @case_id OCP-23745
+  @admin
+  @destructive
+  Scenario: Delete machineautoscaler related machineset doesn't exist
```

Comment:
Prefer a more understandable scenario description like **Machineautoscaler can be deleted when its referenced machineset does not exist**